apps/src/lib/node/ledger/mod.rs

```diff
@@ -249,6 +251,10 @@ async fn run_aux(config: config::Ledger, wasm_dir: PathBuf) {
 
     // Start Tendermint node
     let tendermint_node = start_tendermint(&mut spawner, &config);
+    // wait for genesis time
+    let genesis_time = DateTimeUtc::try_from(config.genesis_time.clone())
+        .expect("Should be able to parse genesis time");
+    sleep_until(genesis_time).await;
```

Comment:
won't this cause issues, since cometbft will try to connect to the abci app and crash? should we sleep up to genesis time before starting cometbft instead?

Reply:
When I tested locally, it seemed fine, but I can easily just move this above the part where we start cometbft